xclim/core/formatting.py

```diff
@@ -321,6 +323,84 @@ def update_history(
     return merged_history
 
 
+def update_xclim_history(func):
```

Comment:
Will this play well with the cf-xarray tracking mechanism ?

Reply:
I haven't studied that thing much, but I feel we are not aiming for the same goal.
`cf-xarray`'s solution is a callback executed _every time_ an _operation_ is performed. 
Here, the history is only updated after a function, so a set of regrouped operation. Something that cf-xarray won't be able to catch.

Will it play well : no. This will _set_ the attribute to something constructed from the call string + the history *of the inputs*. Thus, if cf-xarray modifies anything in the same attribute within the function, that would be discarded.

Reply:
Got it.